libs/utils/include/utils/EntityManager.h

```diff
@@ -23,6 +23,15 @@
 #include <utils/Entity.h>
 #include <utils/compiler.h>
 
+#ifndef FILAMENT_UTILS_TRACK_ENTITIES
+#define FILAMENT_UTILS_TRACK_ENTITIES false
```

Comment:
To prevent code rot, maybe we should default this to `true` in debug builds (i.e. when `NDEBUG` is not defined).

Reply:
The problem is that capturing a stack trace can be very very slow.